protocol/triple/internal/client/cmd/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"dubbo.apache.org/dubbo-go/v3/client"
+	_ "dubbo.apache.org/dubbo-go/v3/imports"
+	greet "dubbo.apache.org/dubbo-go/v3/protocol/triple/internal/proto"
+	"dubbo.apache.org/dubbo-go/v3/protocol/triple/internal/proto/greettriple"
+	"github.com/dubbogo/gost/log/logger"
+)
+
+func main() {
+	//client initialization
+	cli, err := client.NewClient(
+		client.WithURL("127.0.0.1:8080"),
+	)
+	dubboCli, err := greettriple.NewGreetServiceClient(cli)
```

Comment:
Will `GreetService` be better than `GreetServiceClient`?

Reply:
Yes, NewGreetService would be better than NewGreetServiceClient cause two client concepts create confusion.